demo/llm_rag_text_highlight_pdf/run.py

```diff
@@ -0,0 +1,52 @@
+import gradio as gr
```

Comment:
the name is of the demo is kind of long. I think it should just be "highlight_pdf" since that's what it's demonstrating